test/util.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/bash
```

Comment:
It's all good. 

I'll take some time in the next few days and help you clean it up.